File: com-mini-core/src/main/java/com/mini/core/logger/Level.java
package com.mini.core.logger;

/**
 * 日志级别
 * @author xchao
 */
public enum Level {
    ALL(0), TRACE(1), DEBUG(2), INFO(3), WARN(4), ERROR(5), FATAL(6), OFF(7);
    private int value;

    Level(int value) {
        this.value = value;
    }

    public int value() {
        return value;
    }
}

File: sn.mini.kotlin.util/src/main/java/sn/mini/kotlin/util/logger/Level.kt
/**
 * Created the sn.mini.util.logger.Level.java
 * @created 2017年8月14日 下午5:00:39
 * @version 1.0.0
 */
package sn.mini.kotlin.util.logger

/**
 * sn.mini.util.logger.Level.java
 * @author XChao
 */
enum class Level constructor(val value: Int, val out: Int) {
    OFF(0, 0), //
    FATAL(1, 1),//
    ERROR(2, 3),//
    WARN(4, 7),//
    INFO(8, 15), //
    DEBUG(16, 31), //
    TRACE(32, 63),//
    ALL(64, 127) //
}
